### DISCUSSION TITLE
Table toggle column on pivot table

### DISCUSSION THREAD

**Discussion by ahuininga** [filamentphp/filament#5534] [Help]
Hi,

I'm trying to make an toggle column on a pivot table but can't seem to get it working. I've got 2 models App and Modules. One App can have many modules. And a module can belong to many apps. The toggle only gets the right value if I use 'pivot_status' as column value, so the select works. But when I toggle the value I get an sql error.

Column not found: 1054 Unknown column 'pivot_status' in 'field list'
update `modules` SET `pivot_status` = 1

Does anyone know what you need to do to get a toggle column working on a pivot table? Or is it not possible?

Relation on the Module model:

`public function apps()
    {
        return $this->belongsToMany(App::class)->withPivot('status');
    }`

And on the App model

`public function modules()
    {
        return $this->belongsToMany(Module::class);
    }`

My table

    protected function getTableQuery(): Builder
    {
        return Module::query();
    }

    protected function getTableColumns(): array
    {
        return [
            TextColumn::make('name'),
            ToggleColumn::make('pivot_status'),
        ];
    }

**Comment by danharrin** [comment]
you don't need `pivot_status`, if anything it should be just `status` or `pivot.status`
